ReadMe.md

```diff
@@ -27,3 +27,45 @@ License
 -------
 
 QtKeychain is available under the [Modified BSD License](http://www.gnu.org/licenses/license-list.html#ModifiedBSD). See the file COPYING for details.
+
+
+**New:**
+==========
+Added QML support on Qt6 using cmake
+To activate set the qt6 and qml flags
+
+```CMAKE
+-DBUILD_WITH_QT6=ON -DQTKEYCHAIN_BUILD_WITH_QML=ON
+```
+
+```QML
+import QtKeychain 1.0
```

Comment:
Need to think about versioning here. Should it follow the qtkeychain versioning (then this would be 0.15), or not?

Reply:
Gonna set it  to
VERSION "${CMAKE_PROJECT_VERSION_MAJOR}.${CMAKE_PROJECT_VERSION_MINOR}
Becaue the entire QML implementation is in fact the LIbrary so can track the library version.